config/environment.rb

```diff
@@ -1,9 +1,11 @@
-require 'rubygems'
-require 'bundler/setup'
-require 'lotus/setup'
-require_relative '../lib/quizzy'
-require_relative '../apps/web/application'
+require "rubygems"
+require "bundler/setup"
+require "lotus/setup"
+require_relative "../lib/quizzy"
+require_relative "../apps/web/application"
+require_relative "../apps/admin/application"
 
 Lotus::Container.configure do
-  mount Web::Application, at: '/'
+  mount Admin::Application, at: '/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.